Makefile

```diff
@@ -325,7 +379,9 @@ clean:
 	cd Firefox && $(MAKE) clean
 	cd Webmin && $(MAKE) clean
 	cd Chromium && $(MAKE) clean
-	rm -f scap-security-guide.spec
+	rm -f $(RPM_SPEC)
```

Comment:
Hi @konstruktoid. The goal is to support generation of .dsc, .changes and .deb file so that it will be possible, in a future time, to deploy the source repository in the debian repositories to support integration of SSG directly in the distribution. using dpkg-deb, the package will not be integrated in mainstream Debian or Ubuntu.